docker-langserver/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM alpine:3.13.5 as build
+
+RUN apk add --no-cache nodejs npm
+
+RUN npm install -g dockerfile-language-server-nodejs
+
+WORKDIR /code
```

Comment:
this is a different `WORKDIR`, do we want to standardize these?

Reply:
So I think this should be standardized but isn't required here - will remove.